production.ini

```diff
@@ -442,6 +442,15 @@ fedora_epel.mandatory_days_in_testing = 14
 #f28.post_beta.critpath.num_admin_approvals = 0
 #f28.post_beta.critpath.min_karma = 2
 
+###
+### Automatic updates for releases
+###
+
+# You can specify that updates are automatically created for a release when
+# builds are tagged with certain tags. This can be used to enable automatic
+# gating tests which are tied to updates.
+#rawhide.autoupdates.from_tags = f31
```

Comment:
Good, I'd say this resolves all questions I had regarding how to specify from which tags we want to create automatic updates, and into which release. Making it hinge on the candidate tag and a flag rather than being set in configuration makes it less flexible (but this seems an acceptable trade-off) and obviate the code processing configuration.